src/status_im/db.cljs

```diff
@@ -7,13 +7,14 @@
 (def app-db {:contacts/contacts                  {}
              :pairing/installations              {}
              :group/selected-contacts            #{}
+             :ui/old?                            true
```

Comment:
hi @ilmotta, Thank you very much for your PR.
I don't see where we are changing this variable to false, old icon colors is also showing for new ui.

Reply:
Hi @Parveshdhull. I coded under the assumption that the old (current) UI is the only active one and that the new redesign couldn't dynamically be toggled on/off. You commented there's a subscription already for this `status-im.utils.config/new-ui-enabled?`, so I believe I got this part wrong. I should remove this new state instead.